assets/src/js/header.js

```diff
@@ -93,6 +98,20 @@ export const setupHeader = function($) {
     $('.nav-menu-toggle').trigger('click');
   });
 
+  /**
+   * Propagate attributes to all search toggles
+   *
+   * @param {bool} expanded Toggle is expanded
+   */
+  function setSearchToggles(expanded) {
```

Comment:
You can declare this function outside of `setupHeader`. Probably also a few other functions in this file.